ct-app/core/node.py

```diff
@@ -379,9 +374,18 @@ async def relayed_messages_to_db(self):
         Check the inbox for messages.
         """
         messages = []
-        for m in await self.api.messages_pop_all():
+
+        s: socket.socket = self.sockets.get(self.address.id, None)
+
+        if s is None:
+            self.warning(f"No socket for {self.address.id}")
+            return
+
+        for m in s.recv(
+            1024
+        ):  # TODO: check if this is the right way to read from socket and set chunk size wisely
```

Comment:
Pls look into the `tests/test_websocket.py` in the hoprnet monorepo to get the answer.